### ISSUE TITLE
Problem on training

### ISSUE THREAD

**Issue by luigui2906** [charlesCXK/TorchSSC#6]
Hi,

I'm using apex for the first time and I'm experiencing weird behavior. I followed all instructions for the environment installation and to launch the training routine. 

Once all this done I was able to launch the training and the code started running successfully. However, I stopped the training routine and I have not been able to make it run again since. The code starts and blocks 11Mb in both selected GPUs and it stays blocked in there.

Do you have any hint on what could be happening...? 

Thanks in advance for the help and for sharing the great work. Have a nice day.

**Comment by charlesCXK** [comment]
Hi,
According to my previous experience, it is possible that the code was blocked at the beginning of `train.py`, such as some `import` commands. For example, I was once blocked at the command `from seg_opr.sync_bn import DataParallelModel, Reduce, BatchNorm2d` several days ago. Commenting it solves the problem and it will not affect the other part of the code.

If that does not help, you could set breakpoints in the `train.py` and find out which part blocks the running process.

**Comment by luigui2906** [comment]
Hi,

Yep, it was blocked just in there. I was able to solve it. For people that might have the same issue: 

When the training routine starts, a block file is created in the code that it is tried to import (path was `/TorchSSC/furnace/seg_opr/sync_bn/src/cpu` in my case). Since I interrupted the code abruptly, the code is not able to delete this file in order to unblock the folder for a next training. 

Just delete this file and you should be good to go. It worked for me at least. Good luck :).
